linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/H2LoggerConfig.scala

```diff
@@ -0,0 +1,56 @@
+package io.buoyant.linkerd.protocol.h2
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.buoyant.h2.{Request, Response}
+import com.twitter.finagle.{Filter, ServiceFactory, Stack, Stackable}
+import com.twitter.finagle.stack.nilStack
+import io.buoyant.config.PolymorphicConfig
+
+abstract class H2LoggerConfig extends PolymorphicConfig { config =>
```

Comment:
you may have already looked at this, but any thoughts on sharing code with `HttpLoggerConfig`? they're pretty similar, though i imagine the `Request`, `Response` types could be a pain to abstract.

same question for the `IstioLogger` classes.